packages/common/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "tsconfig/nodejs-library.json",
+  "extends": "../../config/tsconfig/nodejs-library.json",
```

Comment:
I think using `../..` is an anti-pattern for this monorepo management tool we are using